WordPress/src/main/java/org/wordpress/android/ui/mysite/items/listitem/SiteListItemBuilder.kt

```diff
@@ -202,8 +202,7 @@ class SiteListItemBuilder @Inject constructor(
     fun buildMeItemIfAvailable(site: SiteModel, onClick: (ListItemAction) -> Unit): ListItem? {
         return if ((!buildConfigWrapper.isJetpackApp &&
                     jetpackFeatureRemovalPhaseHelper.shouldRemoveJetpackFeatures() &&
-                    site.hasCapabilityManageOptions) ||
-            !siteUtilsWrapper.isAccessedViaWPComRest(site)
```

Comment:
We deleted the` !siteUtilsWrapper.isAccessedViaWPComRest(site)` condition from `buildMeItemIfAvailable`. What does this PR make related to the me button?